src/api/api.ts

```diff
@@ -1 +1,7 @@
-export const BASE_URL = 'https://mate.academy/students-api';
+export const BASE_URL = 'https://mate.academy/students-api/';
```

Comment:
I would add that constant to env variables, we can then provide sufficient url for all environments of our api - dev, stage, prod

```suggestion
export const BASE_URL_USERS = `${process.env.BASE_URL}/users`;
```